src/midiEditorCommands.cpp

```diff
@@ -994,6 +994,55 @@ void cmdMidiFilterWindow(Command *command) {
 	}
 }
 
+void maybePreviewCurrentNoteInEventList(HWND hwnd) {
+	auto focused = ListView_GetNextItem(hwnd, -1, LVNI_FOCUSED);
+	char text[50];
+	// Get the text from the type column (4).
+	ListView_GetItemText(hwnd, focused, 4, text, sizeof(text));
+	if (strcmp(text, "Note") != 0) {
+		return;
+	}
+	fill(begin(text), end(text), 0);
```

Comment:
It should be sufficient (and IMO clearer) to do this:
`text[0] = '\0';`

Reply:
Doesn't that only zero the first byte of the string? I"m especially worried about cases where calling ListView_GetItemText writes a string to the string array that's shorter than the string before, which means garbage in it. Or is this fear unfounded?

Reply:
It does only zero the first byte and it does mean there could be garbage for a shorter string. However, if it's writing a shorter string, it should also write out a null terminator (\0), which is required for C strings. So, the only case you have to worry about is where it does nothing, in which case zeroing the first byte will be considered to be an empty C string, since any functions accepting C strings won't go past a null terminator.